src/screens/transaction-address.tsx

```diff
@@ -24,12 +27,10 @@ export const TransactionAddressScreen = () => {
     }
 
     if (!address) {
-      return wallets.snapshot();
+      return wallets;
     }
 
-    return wallets
-      .filtered('address CONTAINS[c] $0 or name CONTAINS[c] $0', address)
-      .snapshot();
+    return wallets.filter(w => w.address === address || w.name === address);
```

Comment:
You should use `.includes` here according to previous implementation

Reply:
Agree. Fixed